src/app/app/multistep/page.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+import OnboardingAboutForm from "@/components/forms/OnboardingAboutForm";
+import OnboardingProfessionalForm from "@/components/forms/OnboardingProfessionalForm";
+import OnboardingContactForm from "@/components/forms/OnboardingContactForm";
+import OnboardingRegistrationForm from "@/components/forms/OnboardingRegistrationForm";
+import { useOnboardingForm } from "@/hooks/useOnboardingForm";
+import { useSearchParams } from "next/navigation";
+
+const steps = [
+  <OnboardingRegistrationForm key={0} />,
+  <OnboardingContactForm key={1} />,
+  <OnboardingProfessionalForm key={2} />,
+  <OnboardingAboutForm key={3} />,
+];
+
+//TODO: Submit go to Next Button
+//TODO: Add persistence and validation to the form
+//TODO: Create Buttons
+//TODO: Design
+//TODO: Transform progress dots into individual component
```

Comment:
I think we can remove some of these comments, as the code already solve some of them

Reply:
Ok, removed. I consider it not to be a good use of TODO, as it pollutes the code.